stdlib/math.jou

```diff
@@ -1,40 +1,112 @@
+# Math lib
+#TODO: add test
+
+# Other functions
+# Returns the absolute value of x.
 declare abs(x: int) -> int
 declare llabs(x: long) -> long
 declare fabs(x: double) -> double
+# Returns x*y+z.
+declare fma(x: double, y: double, z: double) -> double
 
+# Rounding and remainder functions
+# Rounds x upward, returning the smallest integral value that is not less than x.
 declare ceil(x: double) -> double
+# Rounds x downward, returning the largest integral value that is not greater than x.
 declare floor(x: double) -> double
+# Returns the floating-point remainder of numer/denom (rounded towards zero):
+# fmod = numer - tquot * denom
+declare fmod(numer: double, denom: double) -> double
+# Rounds x toward zero, returning the nearest integral value that is not larger in magnitude than x.
+declare trunc(x: double) -> double
+# Returns the integral value that is nearest to x, with halfway cases rounded away from zero.
 declare round(x: double) -> double
+declare lround(x: double) -> long
+# Rounds x to an integral value,
+declare rint(x: int) -> double
+declare lrint(x: long) -> double
+# Rounds x to an integral value
+declare nearbyint(x: double) -> double
+# Returns the floating-point remainder of numer/denom (rounded to nearest):
+# remainder = numer - rquot * denom
+declare remainder(numer: double, denom: double) -> double
 
-declare acos(x: double) -> double
-declare asin(x: double) -> double
-declare atan(x: double) -> double
-declare atan2(y: double, x: double) -> double
+# Trigonometric functions
+# Returns the cosine of an angle of x radians.
 declare cos(x: double) -> double
+# Returns the sine of an angle of x radians.
 declare sin(x: double) -> double
+# Returns the tangent of an angle of x radians.
 declare tan(x: double) -> double
+# Returns the principal value of the arc cosine of x, expressed in radians.
+declare acos(x: double) -> double
+# Returns the principal value of the arc sine of x, expressed in radians.
+declare asin(x: double) -> double
+# Returns the principal value of the arc tangent of, expressed in radians.
+declare atan(x: double) -> double
```

Comment:
Your explanations for math functions make sense for someone who is very familiar with the math, but maybe not for a typical programmer :) We could maybe explain them just a little bit more, so that if you don't quite remember something (e.g. what is a radian or what the 'a' prefix means), that is fine.

```suggestion
# These functions use radians as the angle unit.
# One radian is the angle of a circle slice with equal radius and arc length, about 57 degrees.
# A full turn (360 degrees) is 2pi (about 6.28) radians.
declare cos(x: double) -> double
declare sin(x: double) -> double
declare tan(x: double) -> double
# The 'a' prefixed functions are inverse functions.
# For example, asin is also known as arcsin and sin^-1.
declare acos(x: double) -> double
declare asin(x: double) -> double
declare atan(x: double) -> double
```